contracts/near/eth-client/Cargo.toml

```diff
@@ -21,10 +21,13 @@ ethash = { git = "https://github.com/nearprotocol/rust-ethash" }
 hex = "0.4.0"
 rustc-hex = "2.1.0"
 
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+# serde's Serialize and Deserialize traits are required for `near_bindgen` macro for non-wasm32 targets
+serde = { version = "1.0", features = ["derive"] }
+
```

Comment:
TIL You can have features flags in Cargo.toml

Reply:
@mfornet I didn't check if we can have feature flags there, but I expect this will work only for targets
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#build-dependencies